src/canister/individual_user_template/src/api/post/add_post_v2.rs

```diff
@@ -46,7 +46,7 @@ fn add_post_v2(post_details: PostDetailsFromFrontend) -> Result<u64, String> {
 
     update_scores_and_share_with_post_cache_if_difference_beyond_threshold(&post_id);
 
-    if post_details.creator_consent_for_inclusion_in_hot_or_not {
+    // if post_details.creator_consent_for_inclusion_in_hot_or_not {
```

Comment:
Removed this check which was preventing some timers to get spinned. 

One case that I am unsure about is: If this were the ONLY bug, then there would be NO timers running for certain posts. 

but there are posts / canister_ids where some posts are run, and some are not. 

One check to be performed is : see the status in room_details_map => if there were NO timers started, then ALL of the posts should have BetOngoing for a given canister. 

that is not the case. 

> `dyuzm-uqaaa-aaaal-agt7q-cai/645` has bet ongoing

but  `dyuzm-uqaaa-aaaal-agt7q-cai/656` has bet HotWon status.

Reply:
https://gist.github.com/abeeshake-yral/74345a94b15898a757975517f8180cf3

See this json for reference.

Reply:
so why timers are getting spun for some ? 

because of upgrades, in reenqueue timers, it checks whether posts with hot_or_not_details as Some is true and if yes, they gets spun. and meanwhile when a post is being bet on, hot_or_not_details convert from None to Some. Hence why the resolution is also happening for later slots.

Reply:
mystery solved.